File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        my_hashMap={}
        for i in range(len(nums)):
            if target-nums[i] in my_hashMap:
                return [i,my_hashMap[target-nums[i]]]
            else:
                my_hashMap[nums[i]]=i      
        return[-1,-1]

File: 1-two-sum/two-sum.go
func twoSum(nums []int, target int) []int {
    var mp=map[int]int{}
    for i:=0;i<len(nums);i++{
        if idx,ok := mp[target-nums[i]];ok{
            return []int{idx,i}
        }
       mp[nums[i]]=i
    }
    return []int{-1,-1}
}